index.html

```diff
@@ -6,33 +6,162 @@
+      <ul>
+        <li>Css</li>
+        <li>Html</li>
+        <li>Bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <button type="button" name="See_Project">See Project</button>
+      <div class="cart">
+        <h1>Profesional Art <span>Printing Data</span> </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" name="See-Project">See Project</button>
+      </div>
+      <div class="cart">
+        <h2>Profesional Art <span>Printing Data</span> </h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" name="See-Project">See Project</button>
+      </div>
+       <!--work cart-->
+      <div class="cart">
+        <h1>Profesional Art <span>Printing Data</span> </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" name="See-Project">See Project</button>
+      </div>
+      <div class="cart">
+        <h1>Profesional Art <span>Printing Data</span> </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" name="See-Project">See Project</button>
+      </div>
+      <div class="cart">
+        <h1>Profesional Art <span>Printing Data</span> </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" name="See-Project">See Project</button>
+      </div>
+      <div class="cart">
+        <h1>Profesional Art <span>Printing Data</span> </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" name="See-Project">See Project</button>
+      </div>
+    </section>
+    <!--the about me section-->
+    <footer id="about">
+      <h1>About me</h1>
```

Comment:
- [ ] For best practice, Kindly change the `<h1>` tag here to any of `<h2>` to `<h6>`. It's a good practice to have just one  `<h1>` tag on a webpage for the main title of the page. This will improve the SEO of your page.

Reply:
done too